### ISSUE TITLE
Is there anyone who has achieved the accuracy on rafdb dataset?

### ISSUE THREAD

**Issue by windJRK** [Talented-Q/POSTER_V2#32]
I can only achieve about 91% on radfdb without Imbalaceddatasetsampler.

**Comment by wrr0212** [comment]
Hello, may I ask if you have used his pre-trained model? Without using his pre-trained model, why is my accuracy only 81%? Have you modified any other parameters?

**Comment by windJRK** [comment]
I

> Hello, may I ask if you have used his pre-trained model? Without using his pre-trained model, why is my accuracy only 81%? Have you modified any other parameters?

What is the accuracy you can achieve when you test not train his pre-trained model on RAF-DB dataset?  I get 91% only once without his pre-trained model.

**Comment by wrr0212** [comment]
> 我
> 
> > 您好，请问您是否使用过他的预训练模型？如果不使用他的预训练模型，为什么我的准确率只有 81%？您是否修改了任何其他参数？
> 
> 当您在 RAF-DB 数据集上测试而不是训练他的预训练模型时，您可以达到什么准确性？在没有他的预训练模型的情况下，我只得到了一次 91%。

Did you change any parameters in his code？

**Comment by windJRK** [comment]
> > 我
> > > 您好，请问您是否使用过他的预训练模型？如果不使用他的预训练模型，为什么我的准确率只有 81%？您是否修改了任何其他参数？
> > 
> > 
> > 当您在 RAF-DB 数据集上测试而不是训练他的预训练模型时，您可以达到什么准确性？在没有他的预训练模型的情况下，我只得到了一次 91%。
> 
> Did you change any parameters in his code？

No. Every time I want to change something, the accuracy will only be lower.

**Comment by wrr0212** [comment]
> > > 我
> > > > 您好，请问您是否使用过他的预训练模型？如果不使用他的预训练模型，为什么我的准确率只有 81%？您是否修改了任何其他参数？
> > > 
> > > 
> > > 当您在 RAF-DB 数据集上测试而不是训练他的预训练模型时，您可以达到什么准确性？在没有他的预训练模型的情况下，我只得到了一次 91%。
> > 
> > 
> > Did you change any parameters in his code？
> 
> No. Every time I want to change something, the accuracy will only be lower.

OK, Thanks, I'll try the experiment again

**Comment by asvaaron** [comment]
Hi @windJRK and @wrr0212 , did you manage to fix the following error for RAF-DB database "FileNotFoundError: Couldn't find any class folder in ./data/raf-db/train.". I am using the same suggested folder structure for train and valid?

Do I need any additional folder? Would you please share the link for the RAF-DB dataset you used ?

**Comment by Wenyuzhy** [comment]
Hello, may I ask if you have reproduced the results of his paper? When I used the original configuration in RAF-DB, the reproduction result was only 81%.
